### PR TITLE
Add support for redis sentinel connection

### PR DESCRIPTION
**Pull request by roboxue**
Used a makefile from jedis to setup sentinel servers before test starts

### CODE REVIEW

**Comment by abedra** [comment]
It would be nice to have this, but we do need to sort out how to test it properly before we use it.

**Comment by roboxue** [comment]
Added redis test!

So I copied the `makefile` from Jedis to setup redis sentinel before tests and tear down afterward, and trimmed it to fit our usage pattern here. 

Test commands has been updated to `make test` in .travis.yml

File: src/test/java/com/repsheet/librepsheet/ConnectionTest.java [hunk 1]
```diff
@@ -21,7 +21,6 @@ public void setUp() {
 
     @Test
     public void testConnectionReturnsUsableConnectionPool() {
-        Connection connection = new Connection("localhost");
```
Comment (roboxue) [inline src/test/java/com/repsheet/librepsheet/ConnectionTest.java]: I didn't understand why we need to recreate a connection here given we had it already, so I removed it

**Comment by roboxue** [comment]
The build currently is failing for the cleanup stage when removing /tmp/pid files. Not knowing how to suppress that

**Comment by roboxue** [comment]
Build passed by changing the makefile scripts to do a `ls /tmp/*.pid` loop instead of explicitly listing all the files

**Pull request merged**